gulpfile.js

```diff
@@ -0,0 +1,17 @@
+/*eslint-disable */
+var gulp = require('gulp')
+var gutil = require('gulp-util')
+
+var webpack = require('webpack')
+var webpackConfig = require('./webpack.config')
+
+gulp.task('webpack', function() {
```

Comment:
I think just running webpack might be simpler, it comes with a cli which you could add to package.json/scripts. 
``` js
"build": "webpack",
```
One less dependency?